vendor.conf

```diff
@@ -9,11 +9,11 @@ github.com/moby/locker                              281af2d563954745bea9d1487c96
 github.com/moby/term                                3f7ff695adc6a35abc925370dd0a4dafb48ec64d
 
 # Note that this dependency uses submodules, providing the github.com/moby/sys/mount,
-# github.com/moby/sys/mountinfo, and github.com/moby/sys/symlink modules. Our vendoring
-# tool (vndr) currently does not support submodules / vendoring sub-paths, so we vendor
-# the top-level moby/sys repository (which contains both) and pick the most recent tag,
+# github.com/moby/sys/mountinfo, github.com/moby/sys/signla, and github.com/moby/sys/symlink
+# modules. Our vendoring tool (vndr) currently does not support submodules / vendoring sub-paths,
+# so we vendor the top-level moby/sys repository (which contains both) and pick the most recent tag,
 # which could be either `mountinfo/vX.Y.Z`, `mount/vX.Y.Z`, or `symlink/vX.Y.Z`.
-github.com/moby/sys                                 b0f1fd7235275d01bd35cc4421e884e522395f45 # mountinfo/v0.4.1
+github.com/moby/sys                                 9b0136d132d8e0d1c116a38d7ec9af70d3a59536 # mountinfo/v0.4.1-102-g9b0136d
```

Comment:
Need to tag a release for this; would `v0.5.0` be good for this one? or are we comfortable that `v1.0.0` is ok?

Reply:
0.5.0 is good.

Reply:
v0.5.0 sgtm

Reply:
tagged v0.5.0 👍 https://github.com/moby/sys/releases/tag/signal%2Fv0.5.0